### PR TITLE
Create Python3.py.jinja

### PR DESCRIPTION
**Pull request by liamstaras**
Implement Python template

### CODE REVIEW

File: Python3.py.jinja [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{{ id }}_model = genn_model.create_custom_neuron_class(
```
Comment (tnowotny) [inline Python3.py.jinja:22]: when working with jinja2 in the past I found it useful to add a comment at the top which python variables are expected to be available for use for the template to work, i.e. something like
```
{# Uses variables: parameters, variables, ... #}
```

**Review by tnowotny — COMMENTED** [review]
I would say looks very good overall. See my comment about comment in jinja templates ... one could also envision a minimal comment explaining the purpose of the template ...

File: Python3.py.jinja (hunk 1 above)
Comment (liamstaras) [reply Python3.py.jinja:22]: Yes, good point. Fixed in https://github.com/GenGeNN/GeNN-neuron-lang-bindings/pull/1/commits/7919e3820f2ebc48f536ca4c33cb255b96289312

**Review by tnowotny — APPROVED** [review]
Perfect - thanks alot.

**Review by tnowotny — APPROVED** [review]

**Comment by tnowotny** [comment]
Funny - event hough I have approved twice, it still says "merging is blocked" ...

**Comment by liamstaras** [comment]
Sorted - you didn't have write access. All fixed now.

**Pull request merged**